assets/templates/partials/release/status-header.tmpl

```diff
@@ -34,10 +34,19 @@
           class="ons-u-pb-m"
         {{ end }}
       >
-        <img
-          src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
-          alt="Census 2021 logo"
-        >
+        {{ if eq .Language "en" }}    
```

Comment:
As above on
- trailing spaces 
- switching around so English is the fallback, not the guard

Reply:
this is done now.